extension/main.js

```diff
@@ -16,31 +16,51 @@
 	}
 
 	function update() {
-		gitHubNotifCount(function (count) {
+		gitHubNotifCount(function (count, interval) {
+			var period = 1;
+			if (interval !== parseInt(GitHubNotify.settings.get('interval'), 10)) {
+				GitHubNotify.settings.set('interval', interval);
+				period = Math.ceil(interval / 60);
+				if (period < 1) {
+					period = 1;
+				}
+                chrome.alarms.create({when: Date.now() + 2000 + (period * 60 * 1000)});
```

Comment:
Actually, I take that back. I just noticed the `/60` above.

However, for some reason, my extension doesn't poll every minute now, and I don't know why.